### ISSUE TITLE
Would you update complete mamc loss = L softmax+lambda(L + L + L）？

### ISSUE THREAD

**Issue by carol007** [RQuispeC/npairs_loss_MAMC#1]

**Comment by RQuispeC** [comment]
Hi, I don't have plans to add further implementations to this repo.
For L softmax you can refer to pytorch docs
https://pytorch.org/docs/stable/nn.html?highlight=cross%20entropy#torch.nn.CrossEntropyLoss
